{"name": "test_complete_booking", "status": "broken", "statusDetails": {"message": "AttributeError: 'AssertUtil' object has no attribute 'log'", "trace": "self = <utils.AssertUtil.AssertUtil object at 0x1029913a0>, code = 500, expected_code = 200\n\n    def assert_code(self, code, expected_code):\n        \"\"\"\n        验证返回状态码\n        :param code:\n        :param expected_code:\n        :return:\n        \"\"\"\n        try:\n>           assert int(code) == int(expected_code)\nE           AssertionError\n\n../../utils/AssertUtil.py:21: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\n    def test_complete_booking():\n        # 1、初始化数据库对象\n        conn = init_db(\"db_1\")\n        # 2、查询结果\n        res_db = conn.fetchone(\n            \"select * from shipping_voucher where shipping_document_id  ='52XSeTmQLUC' and type='Booking'\")\n        print(\"数据库查询结果，订舱委托id：\", res_db['id'])\n    \n        # 一、完成「订舱委托」任务\n        booking_url = base_url + \"/shipping_written_material/booking/\" + res_db['id'] + \"?provide_from_order_hub=true\"\n        # 1.读取yaml文件\n        filling_data = YamlReader(\"../../data/filling_booking.yml\").data()\n        # 获取sid\n        cookie = {\n            \"sid\": common.get_sid()\n        }\n        request = Request()\n        r = request.post(booking_url, json=filling_data, cookies=cookie)\n        # print(r)\n>       AssertUtil().assert_code(r[\"code\"], 200)\n\ntest_voucher.py:30: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <utils.AssertUtil.AssertUtil object at 0x1029913a0>, code = 500, expected_code = 200\n\n    def assert_code(self, code, expected_code):\n        \"\"\"\n        验证返回状态码\n        :param code:\n        :param expected_code:\n        :return:\n        \"\"\"\n        try:\n            assert int(code) == int(expected_code)\n            return True\n        except:\n>           self.log.error(\"code error,code is %s,expected_code is %s\" % (code, expected_code))\nE           AttributeError: 'AssertUtil' object has no attribute 'log'\n\n../../utils/AssertUtil.py:24: AttributeError"}, "attachments": [{"name": "log", "source": "ca7d3e90-d3c2-4f80-a20d-afd17e887b49-attachment.txt", "type": "text/plain"}], "start": 1703161777328, "stop": 1703161777847, "uuid": "ff07d5c2-24f1-453c-a9d2-92850dabfca8", "historyId": "66a18088793d7fd43ee8f9e98cd662f1", "testCaseId": "66a18088793d7fd43ee8f9e98cd662f1", "fullName": "testcase.voucher.test_voucher#test_complete_booking", "labels": [{"name": "parentSuite", "value": "testcase.voucher"}, {"name": "suite", "value": "test_voucher"}, {"name": "host", "value": "lifaliangdeAir.lan"}, {"name": "thread", "value": "33935-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.voucher.test_voucher"}]}